pkg/api/handlers/libpod/images.go

```diff
@@ -270,6 +271,18 @@ func ExportImages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// if format is dir and use podman-remote to request, should save to an archive
+	// client will unArchive after receive the archive file
+	// so must convert is at here
+	if query.Remote {
+		if query.Format == define.OCIManifestDir {
```

Comment:
Would it be better to just do this on the client side and not change the server side at all.

Reply:
Server will save to dir, not respone in on file.and client will untar the archive file

Reply:
I am saying that the client checks to see if it is remote and sends across the 

		query.Format = define.OCIArchive
                 or
		query.Format = define.V2s2Archive

And then dumps it locally.

Reply:
> I am saying that the client checks to see if it is remote and sends across the
> 
> ```
> 	query.Format = define.OCIArchive
>              or
> 	query.Format = define.V2s2Archive
> ```
> 
> And then dumps it locally.
if format is an archive, will return at these lines
https://github.com/containers/podman/blob/df6c7c2ae5fb77a557d5616f8b422c45a43fe0ff/pkg/domain/infra/tunnel/images.go#L270-L280
if is a dir, will recreive an archive of the dir, and untar at this line
https://github.com/containers/podman/blob/df6c7c2ae5fb77a557d5616f8b422c45a43fe0ff/pkg/domain/infra/tunnel/images.go#L299
The logic already exists, so I don’t need to add it